lib/puppet/type/keycloak_realm.rb

```diff
@@ -250,4 +250,10 @@ def should_to_s(_newvalue)
   newproperty(:smtp_server_reply_to_display_name) do
     desc 'smtpServer replyToDisplayName'
   end
+
+  newproperty(:brute_force_protected, boolean: true) do
+    desc 'bruteForceProtected'
+    newvalues(:true, :false)
+  end
+
```

Comment:
@aba-rechsteiner If you delete this empty line that should fix the tests. The tests that are failing are rubocop style tests which are rather picky about how things are styled even when functionality is not affected.